tasks/interactive-bots.md

```diff
@@ -16,18 +16,16 @@
 
 ## Background
 
-Zulip [contrib-bots](https://github.com/zulip/zulip/tree/master/contrib_bots/lib)
-is a boilerplate for creating interactive bots that react to messages sent
+The Zulip [contrib-bots](https://github.com/zulip/zulip/tree/master/contrib_bots/lib)
+boilerplate is used for creating interactive bots that react to messages sent
 by users.
 
-The interactive bots live in the `contrib-bots/lib` as python files ('.py') that define
-their specific behavior. The `contrib-bots/run.py` file defines common behaviors
+The interactive bots are constituted of python files ('.py') in `contrib-bots/lib` that define
```

Comment:
* add the sentence "The `contrib-bots/run.py` file defines common behaviors for the interactive bots that react to messages."